labconnect/main/bug_documentation.md

```diff
@@ -0,0 +1,20 @@
+## Bugs that need to be resolved
+
+### Operational Error: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) ambiguous column name: opportunities.id
+
+#### Things We've Attempted & Ruled out
+
+- make & make develop not working
+- the column opportunities.id doesn't exist
+- db doesn't exist
+- new db object (aka it's a different db object)
+- 
+#### ModuleNotFoundError: No module named 'flask_bcrypt'
+ - make: *** [Makefile:11: develop] Error 1
```

Comment:
I explained how to fix this issue. You need to run the `make` command to install the new requirements